day_two_guide/topics/etcd_backup.adoc

```diff
@@ -0,0 +1,1167 @@
+  --initial-advertise-peer-urls *https://192.168.55.8:2380*
+
+2017-10-03 08:55:32.440779 I | mvcc: restore compact to 1041269
+2017-10-03 08:55:32.468244 I | etcdserver/membership: added member 40bef1f6c79b3163 [https://192.168.55.8:2380] to cluster 26841ebcf610583c
+----
+
+. Restore permissions and `selinux` context to the restored files:
++
+----
+# chown -R etcd.etcd /var/lib/etcd/
+# restorecon -Rv /var/lib/etcd
+----
+
+. Start the etcd service:
++
+----
+# systemctl start etcd
+----
+
+. Check for any error messages:
++
+----
+$ journalctl -fu etcd.service
+----
+
+*Adding more nodes*
+
+Once the first instance is running, it is safe to restore multiple etcd servers as desired.
+
+[discrete]
+== Procedure
+
+. Get the etcd name for the instance in the `ETCD_NAME` variable:
++
+----
+# grep ETCD_NAME /etc/etcd/etcd.conf
+----
+
+. Get the IP where etcd listens for peer communication:
++
+----
+# grep ETCD_INITIAL_ADVERTISE_PEER_URLS /etc/etcd/etcd.conf
+----
+
+. Back to the etcd host where etcd is properly running, add the new member:
++
+----
+# etcdctl3 member add *<name>* \
+  --peer-urls="*<advertise_peer_urls>*"
+----
+
+. The command outputs some variables. For example:
++
+----
+ETCD_NAME="master2"
+ETCD_INITIAL_CLUSTER="master-0.example.com=https://192.168.55.8:2380"
+ETCD_INITIAL_CLUSTER_STATE="existing"
+----
++
+Add those values to the `/etc/etcd/etcd.conf` file of the new host:
++
+----
+# vi /etc/etc/etcd.conf
+----
+
+. In the recently added etcd node, clean the etcd data directories to ensure the
+proper backup is restored keeping the running copy:
++
+----
+# mv /var/lib/etcd /var/lib/etcd.old
+# mkdir /var/lib/etcd
+# chown -R etcd.etcd /var/lib/etcd/
+# restorecon -Rv /var/lib/etcd/
+----
++
+or wipe the etcd data directory:
++
+----
+# rm -Rf /var/lib/etcd/*
+----
+
+. Start the etcd service in the recently added etcd host:
++
+----
+# systemctl start etcd
+----
+
+. Check for errors:
++
+----
+# journalctl -fu etcd.service
+----
+
+. Repeat for every etcd node that is required to be added.
+
+. Verify the cluster has been properly set:
++
+----
+# etcdctl3 endpoint health
+https://master-0.example.com:2379 is healthy: successfully committed proposal: took = 1.423459ms
+https://master-1.example.com:2379 is healthy: successfully committed proposal: took = 1.767481ms
+https://master-2.example.com:2379 is healthy: successfully committed proposal: took = 1.599694ms
+
+# etcdctl3 endpoint status
+https://master-0.example.com:2379, 40bef1f6c79b3163, 3.2.5, 28 MB, true, 9, 2878
+https://master-1.example.com:2379, 1ea57201a3ff620a, 3.2.5, 28 MB, false, 9, 2878
+https://master-2.example.com:2379, 59229711e4bc65c8, 3.2.5, 28 MB, false, 9, 2878
+----
+
+[[scaling-etcd]]
+=== Scaling etcd
+
+Scaling the etcd cluster can be performed vertically by adding more resources to
+the etcd hosts, or horizontally by adding more etcd hosts.
+
+Scaling etcd horizontally can be useful to avoid the API and controller services
```

Comment:
I feel like there is a word missing here.  I think it should be "avoid having the API and controller..." but that doesn't work with the 'if etcd was collocated...' -- consider rewording to make clearer